Foundation/NSThread.swift

```diff
@@ -57,18 +57,30 @@ private func NSThreadStart(_ context: UnsafeMutableRawPointer?) -> UnsafeMutable
 }
 
 open class Thread : NSObject {
-    
+    static internal let _mainThread = Thread(thread: pthread_self())
```

Comment:
I have the same question, but test_mainThreadFirstAccess pass so i decided it's ok.
Is there a better way to guarantee initialization on the main thread?